packages/sdk-manager/src/SdkManagerContextProvider.ts

```diff
@@ -0,0 +1,91 @@
+import { Environment } from '@buildwithsygma/sygma-sdk-core';
+import { consume, createContext, provide } from '@lit/context';
+import { LitElement, html } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import {
+  WalletManagerContext,
+  WalletManagerController
+} from '@buildwithsygma/sygmaprotocol-wallet-manager';
+import { SdkManagerState } from './types';
+import { SdkManager } from './SdkManager';
+
+export const SdkManagerContext = createContext<SdkManagerState | undefined>(
+  'sdk-context'
+);
+
+/**
+ * @name SdkManagerContextProvider
+ * @description This component is responsible for providing the SdkManagerController as a context to all its children.
```

Comment:
could you add an example on the implementation, so we have the same as we do in the wallet package?